packages/openactive-integration-tests/test/features/core/common-error-conditions/implemented/incomplete-customer-details-test.js

```diff
@@ -0,0 +1,112 @@
+const { expect } = require('chai');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { FlowHelper } = require('../../../../helpers/flow-helper');
+const { RequestState } = require('../../../../helpers/request-state');
+const { GetMatch, C1, C2, B } = require('../../../../shared-behaviours');
+
+/**
+ * @typedef {import('chakram').ChakramResponse} ChakramResponse
+ */
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'core',
+  testFeature: 'common-error-conditions',
+  testFeatureImplemented: true,
+  testIdentifier: 'incomplete-customer-details',
+  testName: 'Expect an IncompleteCustomerDetailsError when customer details are incomplete',
+  testDescription: 'Run each of C2 and B for a valid opportunity, with customer details incomplete, expecting an IncompleteCustomerDetailsError to be returned (C1 is ignored because customer details are not accepted for C1)',
+  // The primary opportunity criteria to use for the primary OrderItem under test
+  testOpportunityCriteria: 'TestOpportunityBookable',
+  // The secondary opportunity criteria to use for multiple OrderItem tests
+  controlOpportunityCriteria: 'TestOpportunityBookable',
+},
+(configuration, orderItemCriteria, featureIsImplemented, logger) => {
+  /**
+   * Fetch some opportunities and run C1
+   *
+   * Note: This generates jest blocks like `beforeAll()`, `it()`, etc. Therefore, this must be run within a `describe()` block
+   *
+   * @param {RequestState} state
+   * @param {FlowHelper} state
+   */
+  function runC1(state, flow) {
+    beforeAll(async () => {
+      await state.fetchOpportunities(orderItemCriteria);
```

Comment:
I've experimented by not including the `await chakram.wait()` line, which I don't think is needed as we await chakram requests (as far as I'm aware) as soon as they happen. Seems to work 👍

Reply:
Ye think this works if only waiting for one request...